app/javascript/components/recommendations/dropdown-teams.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <froggo-dropdown>
+    <template #btn>
+      <div class="flex items-center justify-between p-3 py-4 mt-1 border rounded-lg">
+        <div v-if="fetchingDefaultPreferences">
+          {{ $i18n.t('message.recommendations.loading') }}
+        </div>
+        <div v-else-if="noTeamsInOrganization">
+          {{ $i18n.t('message.recommendations.noTeams') }}
+        </div>
+        <div v-else>
+          {{ selectedTeam.name }}
+        </div>
+        <inline-svg
+          :src="require('../../../assets/images/chevron-down.svg').default"
+          class="text-black fill-current h-6 w-6 ml-2"
+        />
+      </div>
+    </template>
+    <template #body>
+      <div
+        class="bg-white"
+        slot="body"
+      >
+        <div class="grid grid-rows-auto gap-1 font-medium whitespace-no-wrap">
+          <div
+            v-for="team in selectedOrganizationTeams"
+            :key="team.id"
+            class="p-4"
+            @click="selectTeam(team, 1)"
+          >
+            {{ team.name }}
+          </div>
+        </div>
+      </div>
+    </template>
+  </froggo-dropdown>
+</template>
+<script>
+
+import { mapState } from 'vuex';
+import { LOAD_TEAM, LOAD_RECOMMENDATIONS } from '../../store/action-types';
+import { SET_NO_TEAMS_IN_ORGANIZATION } from '../../store/mutation-types';
+
+export default {
+  props: {
+    teams: {
+      type: Array,
+      default: () => [],
+    },
+    login: {
+      type: String,
+      required: true,
+    },
+  },
+  computed: {
+    ...mapState({
+      selectedTeam: state => state.preferences.team,
+      selectedOrganization: state => state.preferences.organization,
+      fetchingDefaultPreferences: state => state.preferences.fetchingDefaultPreferences,
+      noTeamsInOrganization: state => state.recommendations.noTeamsInOrganization,
+    }),
+    selectedOrganizationTeams() {
+      return this.teams.filter(team => team.organization_id === this.selectedOrganization.id);
+    },
+  },
+  methods: {
+    selectTeam(team) {
+      this.$store.dispatch(LOAD_TEAM, team);
+      this.$store.dispatch(LOAD_RECOMMENDATIONS, this.login);
+    },
+  },
+  watch: {
+    selectedOrganization(organization, oldOrganization) {
+      const teamInOrganization = this.teams.find(team => team.organization_id === organization.id);
```

Comment:
Por qué no ocupo `this.selectedOrganizationTeams`? se ejecuta antes esto?

Reply:
por `oldOrganization` del comentario siguiente.

Reply:
no me queda claro